networkit/csbridge.py

```diff
@@ -0,0 +1,104 @@
+# local imports
+from .support import MissingDependencyError
+
+# external imports
+try:
+	import ipycytoscape
+except ImportError:
+	have_cyto = False
+else:
+	have_cyto = True
+
+try:
+	import seaborn
+except ImportError:
+	have_seaborn = False
+else:
+	have_seaborn = True
+
+def widget_from_graph(G, node_scores = None, node_colors = None, show_ids = True):
+	""" Creates a ipycytoscape-widget from a given graph. The returned widget already contains
+		all nodes and edges from the graph. Optionally the graph can be colored using a seaborn
+		perceptually uniform color map (rocket) or a user given custom color map. See matplotlib
+		color maps for the correct formatting: 
+		https://matplotlib.org/api/_as_gen/matplotlib.colors.Colormap.html#matplotlib.colors.Colormap 
```

Comment:
Indentation seems wrong here.

Reply:
Thanks!

As a side-node: I copied the indentation from other .py-files inside the project. Seems that there some different formats in place in respect to new-lines, indendation, etc. Now adapted the same styling as in the pyx-files. The doc string in these files were refactored when the documentation changed with the new sphinx-version.